### ISSUE TITLE
oxygen bubble distributors not working 

### ISSUE THREAD

**Issue by paulspchelp** [micdoodle8/Galacticraft#2418]
oxygen bubble distributor not working   in  Galacticraft 1.8.9  useing  inf oxygen and  inf battery
[Build 45]

**Comment by SteveKunG** [comment]
I think the problem is Oxygen Storage Module doesn't transfer oxygen into distributor.

![2560-02-26_23 13 52](https://cloud.githubusercontent.com/assets/6128413/23341400/617dfda6-fc79-11e6-8ec8-0e73beaef71e.png)
![2560-02-26_23 13 55](https://cloud.githubusercontent.com/assets/6128413/23341398/61795454-fc79-11e6-878e-71152b396c24.png)
![2560-02-26_23 13 57](https://cloud.githubusercontent.com/assets/6128413/23341399/6179fbac-fc79-11e6-825d-81ecd8fd072c.png)

Oxygen Bubble Distributor still working properly.
![2560-02-26_23 15 15](https://cloud.githubusercontent.com/assets/6128413/23341403/718d8554-fc79-11e6-9bfc-53124cc329cb.png)

**Comment by paulspchelp** [comment]
ass you can see here its not working  for me 
![2017-02-27_14 00 11_2](https://cloud.githubusercontent.com/assets/25328686/23347291/4879005e-fcf5-11e6-8490-8cc5f4392dd3.png)
![2017-02-27_14 00 31](https://cloud.githubusercontent.com/assets/25328686/23347294/4d54423c-fcf5-11e6-81ac-ef8bf0f5b806.png)

**Comment by paulspchelp** [comment]
![2017-02-27_14 04 33](https://cloud.githubusercontent.com/assets/25328686/23347330/c58d0cf2-fcf5-11e6-80ad-44dad28f489c.png)

**Comment by MJRLegends** [comment]
Might be due to the problem i found in the issue i reported about the oxygen system #2387

**Comment by radfast** [comment]
@paulspchelp sorry it's taken me a while to get this one

I can't reproduce this, in SSP, in latest ([build 53](https://forum.micdoodle8.com/index.php?threads/looking-for-1-8-9-1-10-2-1-11-2-beta-testers.6555/) or [build 55](https://wiki.micdoodle8.com/wiki/DownloadTemp) versions of GC 1.8.9 BETA.

Oxygen Bubble Distributor seems to be working 100% OK: inside the bubble the player has oxygen, outside the bubble the oxygen setup is invalid.  Pipes from the Oxygen Storage Module into the Bubble Distributor are working fine.

Are you playing on a server?  Please list Forge version and other mods installed?

**Comment by radfast** [comment]
? is this linked to #1718

**Comment by radfast** [comment]
Closing due to lack of information, can't reproduce, and nobody else has reported ... (or else it is a duplicate of 1718, in which case it can properly be closed).

If it's still an issue seen in recent Galacticraft versions, please comment or re-open, and please give more info.... 

- SSP / SMP / Lan / Sponge?
- Other mods?
- Problem happens all the time or only after a player death?
- etc....
